projects/feature-flag.md

```diff
@@ -0,0 +1,111 @@
+# Feature Flag Semantic Conventions
+
+OpenFeature is a CNCF Incubating project aiming to standardize feature flags.
+They currently provide 11 open source SDKs in various languages and support more than 10 feature flagging backends.
+OpenFeature has had contributions from over 50 companies and is actively developed by a combination of end users, feature flagging vendors, and platform providers.
```

Comment:
Could link to the lists at https://openfeature.dev/community/interested-parties.